docs/source/using-diffusers/loading.mdx

```diff
@@ -12,7 +12,347 @@ specific language governing permissions and limitations under the License.
 
 # Loading
 
-The core functionality for saving and loading systems in `Diffusers` is the HuggingFace Hub.
+A core premise of the diffusers library is to make diffusion models **as accesible as possible**.
+Accessibility is hereby achieved by providing an API to load complete diffusion pipelines as well as individual components with a single line of code. 
+
+In the following we explain in-detail how to easily load:
+
+- *Complete Diffusion Pipelines* via the [`DiffusionPipeline.from_pretrained`]
+- *Diffusion Models* via [`ModelMixin.from_pretrained`]
+- *Schedulers* via [`ConfigMixin.from_config`]
+
+## Loading pipelines
+
+The [`DiffusionPipeline`] class is the easiest way to access any diffusion model thas is [available on the Hub](https://huggingface.co/models?library=diffusers). Let's look at an example on how to download [CompVis' Latent Diffusion model](https://huggingface.co/CompVis/ldm-text2im-large-256).
+
+```python
+from diffusers import DiffusionPipeline
+
+repo_id = "CompVis/ldm-text2im-large-256"
+ldm = DiffusionPipeline.from_pretrained(repo_id)
+```
+
+Here [`DiffusionPipeline`] automatically detects the correct pipeline (*i.e.* [`LDMTextToImagePipeline`]), downloads and caches all required configuration and weight files (if not already done so), and finally returns a pipeline instance, called `ldm`.
+The pipeline instance can then be called using [`LDMTextToImagePipeline.__call__`] for text-to-image generation.
+
+Instead of using the generic [`DiffusionPipeline`] class for loading, one can also directly use the fitting pipeline class. The code snippet above yields the same instance as when doing:
```

Comment:
```suggestion
Instead of using the generic [`DiffusionPipeline`] class for loading, you can also load the appropriate pipeline class directly. The code snippet above yields the same instance as when doing:
```